client/human/HumanClientApp.cpp

```diff
@@ -661,7 +663,22 @@ void HumanClientApp::MultiPlayerGame() {
         m_networking->SendMessage(HostMPGameMessage(server_connect_wnd->GetResult().player_name));
         m_fsm->process_event(HostMPGameRequested());
     } else {
-        m_networking->SendMessage(JoinGameMessage(server_connect_wnd->GetResult().player_name, server_connect_wnd->GetResult().type));
+        boost::uuids::uuid cookie = boost::uuids::nil_uuid();
+        try {
+            if (!GetOptionsDB().OptionExists("network.server.cookie." + server_dest))
+                GetOptionsDB().Add<std::string>("network.server.cookie." + server_dest, "OPTIONS_DB_SERVER_COOKIE", "");
```

Comment:
probably need `UserString` wrapper around `"OPTIONS_DB_SERVER_COOKIE"`, though I don't see it being added to the stringtable, so perhaps not?

Reply:
It shouldn't be accessible from command-line and shoudn't appear in `--help` output so it doesn't needed.